sopel/coretasks.py

```diff
@@ -309,7 +391,10 @@ def retry_join(bot, trigger):
 @module.thread(False)
 @module.unblockable
 def handle_names(bot, trigger):
-    """Handle NAMES response, happens when joining to channels."""
+    """Handle NAMES response, happens when joining to channels.
+
+    This function keeps track of user's privileges.
```

Comment:
```suggestion
    """Handle NAMES responses.

    This function keeps track of users' privileges when Sopel joins channels.
```

+wrap I guess, if needed